src/components/Card/__stories__/CardShowcase.scss

```diff
@@ -0,0 +1,57 @@
+.card-showcase-stories:not(:first-child) {
+    margin-left: 24px;
+}
+
+.card-showcase-stories {
```

Comment:
We shouldn't change styles of the component. Instead we need add content that styled accordingly

Reply:
I've refactored stories 6c24cf8